redfish-core/lib/memory.hpp

```diff
@@ -43,6 +44,10 @@
 namespace redfish
 {
 
+// Interfaces which imply a D-Bus object represents a Memory
+constexpr std::array<const char*, 1> dimmInterfaces = {
+    "xyz.openbmc_project.Inventory.Item.Dimm"};
```

Comment:
It would be better to use `std::string_view` instead of `const char*` for the consistency throughout bmcweb code base

Reply:
done